### PR TITLE
Fixed mixed kotlin and java integration tests

### LINKED ISSUE #1371: Un-ignore mixed sources tests when kotlin 1.1.3 is released.
kotlin 1.1.3 includes support for passing java files. We can un-ignore the ignored mixed sources tests when the final version of the compiler is released.

### PR DESCRIPTION
**Pull request by thalescm**
This fixes #1371 (also you can close #1399 too after).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,7 @@
 /.idea/uiDesigner.xml
 /.idea/workspace.xml
 /.idea/misc.xml
+/.idea/markdown*
```
Comment (styurin) [inline .gitignore]: doesn't look like this change is related, can you revert it?

**Comment by thalescm** [comment]
@styurin :done:

**Comment by styurin** [comment]
There are some problems with formatting. Could you reformat the files? See https://github.com/facebook/buck/blob/master/CONTRIBUTING.md#code-style to find out how to do it.

**Comment by thalescm** [comment]
Huuum, added the pre commit phase, it apparently didn't change anything. @styurin do you know which files were it complaining? I didn't find any that wasn't on the format. Actually if you check the changes, none of them involved any indentation or spacing in source files. The formatting problem seems unrelated to the changes of this PR.
But if there are, if you could point them out it would be a pleasure to fix.

**Comment by thalescm** [comment]
@styurin think I found it (should be fixed in this last commit) ☝️

**Comment by thalescm** [comment]
@styurin anything else I need to change?

**Comment by styurin** [comment]
Nope, it should be merged in later today

**Pull request closed without merging**